plugins/orchestrator-common/src/types.ts

```diff
@@ -65,3 +72,13 @@ export enum WorkflowCategory {
   ASSESSMENT = 'assessment',
   INFRASTRUCTURE = 'infrastructure',
 }
+
+export interface WorkflowOverview {
+  workflowId: string;
+  name?: string;
+  lastTriggered: string | number;
+  lastRunStatus: string;
```

Comment:
For the sake of consistency and readability I suggest that all the fields should be optional, and be undefined instead of contain the empty string ''

Reply:
[231c3da](https://github.com/caponetto/backstage-plugins/pull/36/commits/231c3da23ee8eb404e62d7006bf1364d767ece19)